test/suites/security.sh

```diff
@@ -107,3 +107,25 @@ test_security() {
   LXD_DIR="${LXD_DIR}"
   kill_lxd "${LXD_STORAGE_DIR}"
 }
+
+test_security_protection() {
+  ensure_import_testimage
+  ensure_has_localhost_remote "${LXD_ADDR}"
+
+  lxc launch testimage c1
+  lxc stop c1
+  lxc snapshot c1
+  lxc delete c1
+
+  lxc profile set default security.protection.delete true
```

Comment:
You need to undo this at the end of your test otherwise all subsequent tests will fail to delete (as can be seen on Jenkins).

Reply:
Done.